app/src/main/kotlin/com/wire/android/ui/common/StatusBox.kt

```diff
@@ -46,17 +47,24 @@ import com.wire.android.ui.theme.wireColorScheme
 @Composable
 fun StatusBox(
```

Comment:
suggestion: since it's now used in multiple places, not only for the statuses, maybe we should rename it to be something more like `BadgeBox`? 🤔